flux-refineck/src/checker.rs

```diff
@@ -58,6 +58,8 @@ pub(crate) struct Checker<'ck, 'tcx, M> {
     def_id: DefId,
     /// [`Generics`] of the function being checked.
     generics: Generics,
+    /// [`Expr`]s used to instantiate EarlyBinders for signature of function being checked
+    refparams: Vec<Expr>,
```

Comment:
This should become a list

```suggestion
    refparams: List<Expr>,
```